plugin/src/test/java/org/jenkinsci/plugins/workflow/cps/steps/RestartingLoadStepTest.java

```diff
@@ -1,7 +1,7 @@
 package org.jenkinsci.plugins.workflow.cps.steps;
 
 import edu.umd.cs.findbugs.annotations.CheckForNull;
-import javax.inject.Inject;
+import jakarta.inject.Inject;
```

Comment:
FYI https://github.com/jenkinsci/workflow-cps-plugin/blob/dee06329dfb53029be095128cf5098a64287dc96/plugin/src/test/java/org/jenkinsci/plugins/workflow/cps/steps/RestartingLoadStepTest.java#L39-L40 is a weird old idiom that is completely gratuitous as you can either use `Jenkins.get` or `JenkinsRule.jenkins`.